apis/cloudidentity/v1beta1/membership_types.go

```diff
@@ -32,6 +32,7 @@ type CloudIdentityMembershipSpec struct {
 	GroupRef GroupRef `json:"groupRef"`
 
 	// Immutable. The `EntityKey` of the member. Either `member_key` or `preferred_member_key` must be set when calling MembershipsService.CreateMembership but not both; both shall be set when returned.
+	// Do not set. This is a legacy OUTPUT-ONLY field.
```

Comment:
Could you clarify the consequence of using this field? Without the clarification, users might be concerned that it will error out if it is set. Though it won't and shouldn't for backwards compatibility purposes.

E.g. `This field and its subfields are output-only and should not be set. Values set under this field will be ignored during the reconciliation.`

Reply:
If it'll error out then sure the warning makes sense.